packages/components/select-v2/src/select.vue

```diff
@@ -22,6 +22,8 @@
       trigger="click"
       :persistent="persistent"
       :append-to="appendTo"
+      :show-arrow="showArrow"
+      :offset="!showArrow ? 4 : undefined"
```

Comment:
`show-arrow` should not affect `offest`. We can refer to the [tooltip](https://element-plus.org/en-US/component/tooltip.html#attributes) component and add a new `offset` property with a default value of `12`. The specific value is configurable by the user.

Reply:
OK, I added it. Should I also add some examples in the documentation?